app/views/admin/workshop/_attendances.html.haml

```diff
@@ -23,6 +23,11 @@
               %span.has-tip{ 'data-tooltip': '', 'aria-haspopup': 'true', title: 'Multiple no shows and attendance warnings in the last six months' }
                 %i.fa.fa-exclamation-circle
 
+            - recent_notes = invitation.member.recent_notes
```

Comment:
Can we maybe move this to the member presenter. We should avoid setting up variables in views if possible as they are easy to miss and add unecessary noise..